src/main/java/core/basesyntax/dao/OperationDao.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.dao;
+
+public interface OperationDao {
```

Comment:
Consider renaming the interface 'OperationDao' to better reflect the purpose of the interface, such as 'FruitTransactionDao'.